fre/make/create_compile_script.py

```diff
@@ -1,17 +1,40 @@
 '''
-TODO: make docstring
+Create a compile script to generate a model executable.
 '''
+
 import os
 import logging
-fre_logger = logging.getLogger(__name__)
 
 from pathlib import Path
 from multiprocessing.dummy import Pool
 
-from .gfdlfremake import varsfre, yamlfre, targetfre, buildBaremetal
 import fre.yamltools.combine_yamls_script as cy
+from .gfdlfremake import varsfre, yamlfre, targetfre, buildBaremetal
+
+fre_logger = logging.getLogger(__name__)
 
 def compile_create(yamlfile, platform, target, jobs, parallel, execute, verbose):
```

Comment:
Based on the description below, it seems like some of these arguments can be made optional (parallel: int = 1)